lib/writer.js

```diff
@@ -18,6 +18,8 @@ var TYPES = {
   test: 'Tests'
 };
 
+var PR_REGEX = new RegExp(/#[1-9][\d]*/g);
```

Comment:
Nit pick: could you move this constant in between `DEFAULT_TYPE` and `TYPES` to keep them all together and alphabetized?

Reply:
Done.